src/daemon/rpc.h

```diff
@@ -47,35 +47,41 @@ class t_rpc final
   }
 private:
   cryptonote::core_rpc_server m_server;
+  const std::string m_description;
 public:
   t_rpc(
       boost::program_options::variables_map const & vm
     , t_core & core
     , t_p2p & p2p
+    , const bool restricted
+    , const bool testnet
+    , const std::string port
+    , const std::string description
```

Comment:
Maybe forgot a & ? (unless you know what you're doing with copy elision, I'm not very familiar with those)